.github/workflows/stale.yml

```diff
@@ -13,4 +13,7 @@ jobs:
         stale-issue-label: 'no-issue-activity'
         days-before-stale: 30
```

Comment:
@lt72, please confirm if you want to change days before stale from 30 to 10.  thanks.

Reply:
yes, please